crates/router/src/connector/adyen/transformers.rs

```diff
@@ -167,6 +173,9 @@ impl ForeignFrom<(bool, AdyenStatus)> for storage_enums::AttemptStatus {
             AdyenStatus::Error | AdyenStatus::Refused => Self::Failure,
             AdyenStatus::Pending => Self::Pending,
             AdyenStatus::Received => Self::Started,
+            AdyenStatus::PayoutConfirmReceived => Self::Started,
+            AdyenStatus::PayoutSubmitReceived => Self::Pending,
+            AdyenStatus::PayoutDeclineReceived => Self::Voided,
```

Comment:
We are re-using existing AdyenStatus, since the types are similar for both payments and payouts; although, we don't reuse AttemptStatus for payouts, we need to cover all the mappings. I have added the mappings logically